tests/series/test_properties.py

```diff
@@ -25,58 +39,61 @@
     from pandas.core.indexes.accessors import TimestampProperties  # noqa: F401
 
 
-def test_dt_property() -> None:
+def test_property_dt() -> None:
     """Test the Series.dt property"""
     check(
-        assert_type(pd.Series([pd.Timestamp(2025, 9, 28)]).dt, "TimestampProperties"),
+        assert_type(Series([Timestamp(2025, 9, 28)]).dt, "TimestampProperties"),
         DatetimeProperties,
     )
     check(
-        assert_type(pd.Series([pd.Timedelta(1, "s")]).dt, TimedeltaProperties),
+        assert_type(Series([Timedelta(1, "s")]).dt, TimedeltaProperties),
         TimedeltaProperties,
     )
     check(
         assert_type(
-            pd.period_range(start="2022-06-01", periods=10).to_series().dt,
+            period_range(start="2022-06-01", periods=10).to_series().dt,
             PeriodProperties,
         ),
         PeriodProperties,
     )
 
     if TYPE_CHECKING_INVALID_USAGE:
-        s = pd.DataFrame({"a": [1]})["a"]
+        s = DataFrame({"a": [1]})["a"]
         # python/mypy#19952: mypy believes Properties and its subclasses have a
         # conflict and gives Any for s.dt
         assert_type(s.dt, Properties)  # type: ignore[assert-type]
-        _1 = pd.Series([1]).dt  # type: ignore[arg-type] # pyright: ignore[reportAttributeAccessIssue]
+        _1 = Series([1]).dt  # type: ignore[arg-type] # pyright: ignore[reportAttributeAccessIssue]
 
 
-def test_array_property() -> None:
+def test_property_array() -> None:
     """Test that Series.array returns ExtensionArray and its subclasses"""
+    check(
+        assert_type(Series([1], dtype="category").array, Categorical), Categorical, int
+    )
+    # cast will be removed if pandas-dev/pandas-stubs#1395 is resolved
     check(
         assert_type(
-            pd.Series([1], dtype="category").array,
-            pd.Categorical,
+            cast("Series[Int64Dtype]", Series([1, NA], dtype="Int64")).array,
```

Comment:
I think we should not have a test like this for 2 reasons:

1. It encourages the use of `Series[Int64Dtype]` and I just don't think we would introduce that.  Also, then `Int64Dtype` should really be in `S1`
2. We can have a test with `dtype="Int64"`, but then the results will just be an `ExtensionArray` for now, and if someone wants the real type, they can cast the result.